test/integration/users.test.js

```diff
@@ -350,6 +350,134 @@ describe("Users", function () {
     });
   });
 
+  describe("GET /users?search", function () {
+    const userSearchParamsData = [
+      {
+        username: "ankur",
+        first_name: "Ankur",
+        last_name: "Narkhede",
+        yoe: 0,
+        img: "./img.png",
+        linkedin_id: "ankurnarkhede",
+        github_id: "ankur1234",
+        github_display_name: "ankur-xyz",
+        phone: "1234567890",
+        email: "abc@gmail.com",
+      },
+      {
+        username: "23ankur",
+        first_name: "Ankur",
+        last_name: "Narkhede",
+        yoe: 0,
+        img: "./img.png",
+        linkedin_id: "ankurnarkhede",
+        github_id: "ankur1234",
+        github_display_name: "ankur-xyz",
+        phone: "1234567890",
+        email: "abc@gmail.com",
```

Comment:
This is already available in fixtures, so we should be using it from there, If not available move this to fixtures

Reply:
done